db/migrate/20220123013256_create_teams.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTeams < ActiveRecord::Migration[6.1]
+  def change
+    create_table :teams do |t|
+      t.string :token
```

Comment:
access token と invitation token のどちらも token でミスリーディングしやすいので、コード全体でつねに prefix するようにしておいたほうが良いかもしれません。

```suggestion
      t.string :invitation_token
```

Reply:
確かに二種類のtokenがあるのでそうした方がよさそうですね...！！ご指摘ありがとうございます✨